src/openklant/components/klantinteracties/admin/partijen.py

```diff
@@ -34,34 +33,48 @@ def clean(self):
 
         voorkeurs_digitaal_adres = cleaned_data.get("voorkeurs_digitaal_adres")
         voorkeurs_rekeningnummer = cleaned_data.get("voorkeurs_rekeningnummer")
-
-        if voorkeurs_digitaal_adres:
-            if (
-                voorkeurs_digitaal_adres
-                not in DigitaalAdres.objects.filter(partij=self.instance).all()
-            ):
-                raise ValidationError(
-                    {
-                        "voorkeurs_digitaal_adres": _(
-                            "Het voorkeurs adres moet een gelinkte digitaal adres zijn."
+        soort_partij = self.data.get("soort_partij")
+        identificatie = self.data.get(f"{soort_partij}-TOTAL_FORMS", "")
+
+        if soort_partij and identificatie in ["0", ""]:
```

Comment:
Similar comment as in the other file: it should be checked if `partijIdentificatie` should be a required field, because currently it isn't according to the API spec